mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -209,7 +219,13 @@ class LinearRegression(override val uid: String)
 
     if (handlePersistence) instances.unpersist()
 
-    copyValues(new LinearRegressionModel(uid, weights, intercept))
+    val model = copyValues(new LinearRegressionModel(uid, weights, intercept))
```

Comment:
The earlier `copyValues` can be removed however since I'm not using `model.transform` on that one, but I'm inclined to leave it just for consistency.